src/Microsoft.DotNet.Compiler.Common/Executable.cs

```diff
@@ -133,15 +130,30 @@ private void WriteRuntimeConfig(LibraryExporter exporter)
                 var runtimeOptions = new JObject();
                 json.Add("runtimeOptions", runtimeOptions);
 
-                var redistExport = exporter
-                    .GetAllExports()
-                    .FirstOrDefault(l => l.Library.Identity.Name.Equals(RedistPackageName, StringComparison.OrdinalIgnoreCase));
-                if (redistExport != null)
+                var redistPackage = _context.RootProject.Dependencies
+                    .Where(r => r.Type.Equals(LibraryDependencyType.Platform))
+                    .ToList();
+                if(redistPackage.Count > 0)
                 {
-                    var framework = new JObject(
-                        new JProperty("name", redistExport.Library.Identity.Name),
-                        new JProperty("version", redistExport.Library.Identity.Version.ToNormalizedString()));
-                    runtimeOptions.Add("framework", framework);
+                    if(redistPackage.Count > 1)
+                    {
+                        throw new InvalidOperationException("Multiple packages with type: \"platform\" were specified!");
```

Comment:
I feel this is not right place for this check. Maybe LibraryManager so VS can show it as error in project.json?